File: Assignment_39/Assignment_39_5.c
#include<stdio.h>

void Display(int iNo)
{
    int iCnt = 0;
    char ch = '\0';
    //O(n) 
    for(iCnt = 0,ch = 'a'; iCnt < iNo ; iCnt++, ch++)
    {
        printf("%c\t",ch);
    }
     printf("\n");
}

// Main Function
int main()
{

    int iValue = 5;

    Display(iValue);

    return 0;
}

File: Assignment_39/Assignment_39_5.java
class Demo
{
    public void display(int iNo)
    {
        int iCnt = 0;
        char ch ='\0';
        //O(n) 
        for(iCnt = 0, ch = 'a'; iCnt < iNo ; iCnt++,ch++)
        {
            System.out.printf(ch+"\t");
        }
        System.out.println("");
    }
}

class Assignment_39_5 
{
    public static void main(String A[])
    {
        int iValue = 5;

        Demo dobj = new Demo();

        dobj.display(iValue);
    }
    
}
